aws/field/field.go

```diff
@@ -31,16 +31,16 @@ const (
 	CacheEngine // Cache Engine
 
 	// RDS fields
-	DatabaseEngine   // Database Engine
-	DatabaseEdition  // Database Edition
-	DeploymentOption // Deployment Option
-	LicenseModel     // License Model
+	DatabaseEngine           // Database Engine
+	DatabaseEdition          // Database Edition
+	DatabaseDeploymentOption // Deployment Option
+	LicenseModel             // License Model
 
 	// FSx fields
-	FileSystemType     // File system type
-	StorageType        // Storage type
-	ThroughputCapacity // Throughput capacity
-	Deployment_option  // Deployment option
+	FileSystemType             // File system type
+	StorageType                // Storage type
+	ThroughputCapacity         // Throughput capacity
+	FileSystemDeploymentOption // Deployment option
```

Comment:
we kept as close as possible `Deployment_option  // Deployment option` both name because this is what the user has to search in the csv to find correspondence

But if we want to change it, we will have to re-injest price and change query such: https://github.com/cycloidio/terracost/blob/master/aws/terraform/fsx_x_file_system.go#L129

Reply:
The text description should not have changed, and test are passing (or at least not failing for this reason).

But if you think it should stay the same I can try to fix the linter instead.

Reply:
So remove the change or not?

Reply:
Ah no you are right, we do a matching to the DB field here `field.FileSystemDeploymentOption: "Deployment_option",`
So yes we can change it ;)

Reply:
Ohhh also the linter is the old deprecated one and the new one is finding some good stuff:
```
util/fs.go:19:15: Error return value of `os.MkdirAll` is not checked (errcheck)
			os.MkdirAll(tmppath, info.Mode())
			           ^
util/fs.go:32:10: Error return value of `io.Copy` is not checked (errcheck)
		io.Copy(f, af)
		       ^
util/fs.go:49:16: Error return value of `afs.MkdirAll` is not checked (errcheck)
			afs.MkdirAll(tmppath, d.Type())
			            ^
util/fs.go:62:10: Error return value of `io.Copy` is not checked (errcheck)
		io.Copy(f, osf)
		       ^
testutil/azure_server.go:28:10: Error return value of `w.Write` is not checked (errcheck)
		w.Write(b)
		       ^
testutil/google_server.go:33:10: Error return value of `w.Write` is not checked (errcheck)
		w.Write(b)
		       ^
terraform/hcl.go:255:13: Error return value of `fs.MkdirAll` is not checked (errcheck)
	fs.MkdirAll(dir, 0700)
	           ^
terraform/schema.go:53:18: Error return value of `json.Unmarshal` is not checked (errcheck)
			json.Unmarshal(bv, &e)
			              ^
google/terraform/compute_instance.go:17:2: field `instanceType` is unused (unused)
	instanceType string
	^
aws/terraform/fsx_x_file_system.go:21:2: field `deploymentType` is unused (unused)
	deploymentType               string
	^
azurerm/ingester_test.go:27:7: S1005: unnecessary assignment to the blank identifier (gosimple)
		for _ = range i.Ingest(ctx, 10) {
		    ^
azurerm/ingester_test.go:40:7: S1005: unnecessary assignment to the blank identifier (gosimple)
		for _ = range i.Ingest(ctx, 10) {
		    ^
google/ingester_test.go:31:7: S1005: unnecessary assignment to the blank identifier (gosimple)
		for _ = range i.Ingest(ctx, 10) {
		    ^
terraform/hcl.go:38:28: S1019: should use make([]string, 0) instead (gosimple)
	modules := make([]string, 0, 0)
	                          ^
terraform/hcl.go:343:35: S1019: should use make([]interface{}, 0) instead (gosimple)
				values := make([]interface{}, 0, 0)
				                              ^
terraform/hcl.go:368:30: S1019: should use make([]string, 0) instead (gosimple)
						vars := make([]string, 0, 0)
						                       ^
terraform/hcl.go:390:28: S1019: should use make([]string, 0) instead (gosimple)
				vars := make([]string, 0, 0)
				                       ^
terraform/plan.go:296:39: S1019: should use make([]interface{}, 0) instead (gosimple)
			values[name] = make([]interface{}, 0, 0)
			                                   ^
terraform/plan.go:314:31: S1019: should use make([]interface{}, 0) instead (gosimple)
			refs = make([]interface{}, 0, 0)
			                           ^
aws/terraform/efs_file_system.go:169:51: S1025: the argument is already a string, there's no need to use fmt.Sprintf (gosimple)
				{Key: "UsageType", ValueRegex: util.StringPtr(fmt.Sprintf("%s", usagetype))},
				                                              ^
usage/usage.go:47:2: structtag: struct field tag `json:"resource_default_type_usage",yaml:"resource_default_type_usage"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces (govet)
	ResourceDefaultTypeUsage map[string]interface{} `json:"resource_default_type_usage",yaml:"resource_default_type_usage"`
	^
terraform/schema.go:7:2: structtag: struct field tag `json:"constant_value",mapstructure:"constant_value"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces (govet)
	ConstantValue string   `json:"constant_value",mapstructure:"constant_value"`
	^
terraform/schema.go:8:2: structtag: struct field tag `json:"references",mapstructure:"references"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces (govet)
	References    []string `json:"references",mapstructure:"references"`
	^
e2e/aws_estimation_test.go:217:6: ineffectual assignment to r (ineffassign)
					r, ok := values["region"]
					^
util/fs.go:15:66: SA4009: argument err is overwritten before first use (staticcheck)
	err := afero.Walk(afs, fspath, func(p string, info fs.FileInfo, err error) error {
	                                                                ^
util/fs.go:22:3: SA4009(related information): assignment to err (staticcheck)
		f, err := os.Create(tmppath)
		^
util/fs.go:45:64: SA4009: argument err is overwritten before first use (staticcheck)
	err := filepath.WalkDir(ospath, func(p string, d fs.DirEntry, err error) error {
	                                                              ^
util/fs.go:52:3: SA4009(related information): assignment to err (staticcheck)
		f, err := afs.Create(tmppath)
```